py/mpconfig.h

```diff
@@ -1160,6 +1160,11 @@ typedef double mp_float_t;
 #define MICROPY_PY_MATH_MODF_FIX_NEGZERO (0)
 #endif
 
+// Whether to provide fix for pow(1, NaN), which should be 1 not NaN.
```

Comment:
Perhaps the comment here should match that in modmath, eg "Whether to provide a fix for pow(1, NaN) and pow(NaN, 0), which should both be 1"

Reply:
Spot on, I wrote this before I found the other bug